introduction.md

```diff
@@ -3,3 +3,7 @@ Introduction To Repo
 1)New to world of GitHub.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?